package.json

```diff
@@ -19,9 +19,9 @@
     "dependencies": {
         "@pxblue/colors": "^3.0.0",
         "@pxblue/icons-svg": "^1.2.0",
-        "@pxblue/react-native-components": "6.0.0-alpha.3",
+        "@pxblue/react-native-components": "6.0.1-beta.0",
```

Comment:
Do you want to wait until the official 6.0.1 is out so you don't have to do another PR?